SystemServices/SystemServices.h

```diff
@@ -135,7 +135,8 @@ namespace WPEFramework {
                 void DeinitializeIARM();
 #endif /* defined(USE_IARMBUS) || defined(USE_IARM_BUS) */
 
-                /* Events : Begin */
+                /* Events : Begin */ 
+                int FwUpdateState_LatestEvent;
```

Comment:
Pls change this to begin with "m_" to indicate it's a member. Also pls make it private.